platformLibs/src/platform/osx/builtin.def

```diff
@@ -1,5 +1,6 @@
+depends = posix
```

Comment:
How come?

Reply:
Automatically :) The `set_depends` utility groups all special declarations added by Clang into each translation unit into a separate virtual header. Our current convention assumes that this virtual header is owned by the `posix` library and all other libraries (e.g. `darwin` and all new ones) import it. Thus this dependency appears.

Reply:
I think dep shall be reverted and posix depend on this one, not vice versa.

Reply:
It makes sense but I'm not sure if such dependency change can affect users or not. @SvyatoslavScherbina WDYT?

Reply:
I think that this dependency is mostly technical, and it is somewhat easier to keep it in current state.

Reply:
OK, let's go the current way.